debian/cocaine-v12-runtime.init

```diff
@@ -14,7 +14,7 @@
 # Author: Andrey Sibiryov <kobolog@yandex-team.ru>
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
-NAME=cocaine-runtime
+NAME=cocaine-v12-runtime
```

Comment:
And oh, it reminds me of a story with elliptics and elliptics-2.10(elliptics-2.10=2.23.1.2.3, lol). Go to the next room and ask Andrew Godin about the PAIN if you don't believe me.

The easiest and not so dirty solution is to distribute apt pinning settings together with cocaine-runtime v0.11 package.